aiosmtpd/tests/test_smtp.py

```diff
@@ -1386,6 +1396,30 @@ def test_exception_handler_exception(self):
                          b'Error: (ValueError) ErroringErrorHandler test')
         self.assertIsInstance(handler.error, ValueError)
 
+    def test_exception_handler_multiple_connections_lost(self):
+        handler = ErroringHandlerConnectionLost()
+        controller = Controller(handler)
+        controller.start()
+        self.addCleanup(controller.stop)
+        with SMTP(controller.hostname, controller.port) as client1:
+            code, response = client1.ehlo('example.com')
+            self.assertEqual(code, 250)
+            with SMTP(controller.hostname, controller.port) as client2:
+                code, response = client2.ehlo('example.com')
+                self.assertEqual(code, 250)
+                with self.assertRaises(SMTPServerDisconnected) as cm:
+                    mail = CRLF.join(['Test', '.', 'mail'])
+                    client2.sendmail('anne@example.com', ['bart@example.com'], mail)
```

Comment:
According to `tox -e qa` this line is too long. If you can chop it into two lines, it will pass Travis checking.